universal-login-react/src/ui/Onboarding/ConfirmCodeScreen.tsx

```diff
@@ -19,6 +20,8 @@ export const ConfirmCodeScreen = ({hideModal, walletService}: ConfirmCodeScreenP
   >
     <ConfirmCode
       email={'email@address.com'}
```

Comment:
we should get email from current WalletState

Reply:
true 👍